src/operation-node/drop-type-node.ts

```diff
@@ -2,12 +2,13 @@ import { freeze } from '../util/object-utils.js'
 import { OperationNode } from './operation-node.js'
 import { SchemableIdentifierNode } from './schemable-identifier-node.js'
 
-export type DropTypeNodeParams = Omit<Partial<DropTypeNode>, 'kind' | 'name'>
+export type DropTypeNodeParams = Omit<Partial<DropTypeNode>, 'kind' | 'names'>
 
 export interface DropTypeNode extends OperationNode {
   readonly kind: 'DropTypeNode'
-  readonly name: SchemableIdentifierNode
+  readonly names: SchemableIdentifierNode[]
```

Comment:
This is correct. But how can we make it not break any existing compilers and plugins?